client/components/data/purchases/edit-card-details/index.jsx

```diff
@@ -7,47 +7,34 @@ import React from 'react';
  * Internal dependencies
  */
 import CountriesList from 'lib/countries-list';
-import { fetchStoredCards, fetchUserPurchases } from 'lib/upgrades/actions';
+import { fetchStoredCards } from 'lib/upgrades/actions';
 import observe from 'lib/mixins/data-observe';
-import PurchasesStore from 'lib/purchases/store';
-import { shouldFetchUserPurchases } from 'lib/purchases';
```

Comment:
It looks like we can also retire this `shouldFetchUserPurchases()` function now.